quickfixj-perf-test/pom.xml

```diff
@@ -0,0 +1,71 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.quickfixj</groupId>
+        <artifactId>quickfixj-parent</artifactId>
+        <version>3.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quickfixj-perf-test</artifactId>
+    <packaging>jar</packaging>
+
+    <name>QuickFIX/J Performance test</name>
+    <description>QuickFIX/J Performance test for regression and improvements</description>
+    <url>http://www.quickfixj.org</url>
+
+    <properties>
+        <mainClass>org.openjdk.jmh.Main</mainClass>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.quickfixj</groupId>
+            <artifactId>quickfixj-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.quickfixj</groupId>
+            <artifactId>quickfixj-messages-all</artifactId>
+            <version>3.0.0-SNAPSHOT</version>
```

Comment:
This should probably be the `project.version` ?!

Reply:
Done, thank you for pointing.